rutil/Data.cxx

```diff
@@ -1051,6 +1061,18 @@ Data::c_str() const
    return mBuf;
 }
 
+std::string Data::toString()
```

Comment:
The `operator std::string() const` might be used to make it smooth.
The same point is for `operator std::string_view() const`.

Reply:
OK, updated.

Reply:
I've checked this file, it seems the const presents there:
https://github.com/resiprocate/resiprocate/pull/382/commits/16de3ebbe7fc205aa1daa25b8bd2acb1f01288b2#diff-132e4dc259765215ed262b1f6397093e2e6e64d9c4e2482e41512206a7349781

Maybe due to your cache?

Reply:
My question was about `operator std::string() const` but not about `Data& operator=(const std::string& str)`.
https://en.cppreference.com/w/cpp/language/cast_operator
It could replace `std::string Data::toString()` or at least extend the current interface.

Reply:
Thanks, updated.